python/cugraph/centrality/betweenness_centrality_wrapper.pyx

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019, NVIDIA CORPORATION.
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
why not 2019-2020 like the rest of the header blocks?

Reply:
This was a copy/paste error previously and I noticed it now.  The betweenness centrality code was new in 2020.  I thought the convention was to copyright new code in the year that it was created and to update the copyright when you change existing code.